keel-core/src/test/kotlin/com/netflix/spinnaker/keel/actuation/CheckSchedulerTests.kt

```diff
@@ -21,6 +23,7 @@ internal object CheckSchedulerTests : JUnit5Minutests {
 
   private val resourceRepository = mockk<ResourceRepository>()
   private val deliveryConfigRepository = mockk<DeliveryConfigRepository>()
+  private val combinedRepository: CombinedRepository = combinedInMemoryRepository(deliveryConfigRepository = deliveryConfigRepository, resourceRepository = resourceRepository)
```

Comment:
It seems weird that the test has to rely on all these things. It would make more sense to just mock the combined repository and ignore the underlying implementation.